.circleci/config.yml

```diff
@@ -9,15 +9,12 @@ jobs:
       - node/install:
           install-npm: false
           install-yarn: false
-          node-version: '12.10.0'
+          node-version: '13.8.0'
```

Comment:
I always wonder if there is a way to get the value from `.nvmrc`? I guess the answer is no?

Reply:
Yeah agreed, that would be nice. I think we just have to keep an eye on Circle's Node Orb, and perhaps it'll get smart enough to do this. I notice they do mention nvm in its source code, but don't seem to be using it in the way you suggest. Perhaps we could make a contribution?

https://circleci.com/orbs/registry/orb/circleci/node

Reply:
Sounds good, can't find the repo though 🤔 Would be good to file an issue before contributing